grpc_codegen/avs.proto

```diff
@@ -126,6 +126,7 @@ message ContractWriteNode {
 
   // abi is necessary to decode the return
   string contract_abi = 3;
+  string method = 4;
```

Comment:
Is this new field reflected in the sdk-js source code yet?

We need to make sure all the changes in avs.proto are factored into the source code.